modules/extensions-spring-support/src/test/java/org/mule/runtime/module/extension/internal/resources/ExtensionModelJsonGeneratorTestCase.java

```diff
@@ -54,31 +56,56 @@ public class ExtensionModelJsonGeneratorTestCase extends FileGenerationParameter
+                        newUnitTest(JAVA_LOADER, TransactionalExtension.class, "tx-ext.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "mule-tx-extension", VERSION)),
+                        newUnitTest(JAVA_LOADER, SubTypesMappingConnector.class, "subtypes.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "mule-subtypes-extension", VERSION)),
+                        newUnitTest(JAVA_LOADER, MarvelExtension.class, "marvel.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "mule-marvel-extension", VERSION)),
+                        newUnitTest(SOAP_LOADER, RickAndMortyExtension.class, "ram.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "mule-rick-and-morty-extension", VERSION)),
+                        newUnitTest(JAVA_LOADER, TypedValueExtension.class, "typed-value.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "mule-typed-value-extension", VERSION)),
+                        newUnitTest(JAVA_LOADER, TestOAuthExtension.class, "test-oauth.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "mule-test-oauth-extension", VERSION)),
+                        newUnitTest(JAVA_LOADER, WeaveFunctionExtension.class, "test-fn.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "test-weave-function-extension", VERSION)),
+                        newUnitTest(JAVA_LOADER, ValuesExtension.class, "values.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "mule-values-extension", VERSION)),
+                        newUnitTest(JAVA_LOADER, SampleDataExtension.class, "sample-data.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "mule-sample-data-extension", VERSION)),
+                        newUnitTest(JAVA_LOADER, ImplicitConfigExtension.class, "implicit-config.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "mule-implicit-config-extension", VERSION)),
+                        newUnitTest(JAVA_LOADER, NonImplicitConfigExtension.class, "non-implicit-config.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "mule-non-implicit-config-extension", VERSION)),
+                        newUnitTest(JAVA_LOADER, SemanticTermsExtension.class, "semantic-terms-extension.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "mule-semantic-terms-extension", VERSION)),
+                        newUnitTest(JAVA_LOADER, ReconnectionExtension.class, "reconnection-extension.json",
+                                    createArtifactCoordinate(TEST_GROUP_ID, "mule-reconnection-test-extension", VERSION)));
 
     return createExtensionModels(extensions);
   }
 
+  private static ArtifactCoordinates createArtifactCoordinate(String groupId, String artifactId, String version) {
```

Comment:
groupId and version are always the same. I think the code calling this will be more readable if those are implicit